packages/database/src/object-list.ts

```diff
@@ -1,23 +1,22 @@
 import {objectLists, objectItems} from './db/schema';
 import {insertObjectItemSchema, insertObjectListSchema} from './db/validation';
-import {InferSelectModel, sql} from 'drizzle-orm';
+import {sql} from 'drizzle-orm';
 import {db} from './db/connection';
 import {iriToHash} from './iri-to-hash';
 import {DBQueryConfig, eq} from 'drizzle-orm';
+import {SelectObjectList} from './db/types';
 
 interface Option {
   withObjects?: boolean;
   limitObjects?: number;
-}
-
-interface ObjectList extends InferSelectModel<typeof objectLists> {
-  objects?: InferSelectModel<typeof objectItems>[];
+  includeObjectIri?: string;
```

Comment:
Minor: the word `include` could be interpreted as being a boolean (see e.g. JS's `Array.includes()`). Do you see a way to omit the word or replace it with something else?

Reply:
I will just call it `objectIri`